web/packages/core/src/ruffle-player.ts

```diff
@@ -1991,21 +1994,7 @@ export class RufflePlayer extends HTMLElement {
     }
 
     protected debugPlayerInfo(): string {
```

Comment:
Is it possible to have a `RufflePlayer` instance that is not also a `RuffleObject` instance or some other instance of a subtype? If so, I wonder if it might be nice to have some sort of string here that describes what it is (like `"Player type: Object\n"` in ruffle-object.ts). But, https://github.com/ruffle-rs/ruffle/blob/27738a492e3ea01d9e5fdd5a99147a15dac422ed/web/packages/core/src/ruffle-object.ts#L173 calls `super.debugPlayerInfo();`. I do not know what would be best here, but, your changes here are overall good as far as I can tell.

(I personally like to avoid inheritance, and prefer composition and the like).